<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputEkstraksiDanEnkripsi</class>
 <widget class="QDialog" name="InputEkstraksiDanEnkripsi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input Ekstraksi dan Dekripsi</string>
  </property>
  <widget class="QLabel" name="lblNilaiX0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nilai X0 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNilaiY0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nilai Y0 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDcmatrix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DC Matrix      :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInputKunciX0Y0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>125</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input Kunci X0 dan Y0</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxX0">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000001000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxY0">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000001000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKembali">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>610</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kembali</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCitraStegano">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="citraSteganoPath">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDcMatrix">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>771</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dcMatrixPath">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblCitraStegano">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Citra Stegano :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEkstraksiDanDekripsi">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ekstraksi dan Enkripsi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
